### PR TITLE
rewrite learning page FAQ #3246

### LINKED ISSUE #3246: Remove reference to stability without stagnation
This section in the FAQ dates us a lot.

I'll draft something else.

### PR DESCRIPTION
**Pull request by jenweber**
Resolves #3246 

I added some things... For convenience of review:

### What is Glimmer? Do I need to learn it too?
Glimmer is a rendering engine used in Ember. However, in order to build Ember apps, you don't need to know anything about Glimmer, other than it's super fast and that the contributors to the core codebase love to geek out about it. It's all behind the scenes. If you're curious to know how the Glimmer project fits into the roadmap for Ember's development, you can learn more here.

### How do I pitch Ember at my company?
Here are some strategies that may help you out. (link to reader questions)
 
### Is it ok to use controllers?
Yes. Controllers are the right place to put URL query params, actions and attributes that need to be passed from the route level to components, and lifecycle hook code.

### How do I future-proof my Ember.js application?

Ember.js has a strong emphasis on developer productivity, which means that framework developers take great care when designing new functionality. Here are some tips, tools, and processes that help you keep your apps easy to update, so you can take advantage of new features without breaking your existing work.

- Feature Flags: Many new features are implemented under optional flags so that users can try them out and report any issues.
- RFCs: Ember.js follows a community-driven process for its development that incorporates RFCs for features and deprecations. An RFC is a Request for Comments. Before changes are made to Ember's API, they are proposed publicly and the community is asked to review and comment. Anyone can create an RFC, not just Core Team members. RFCs undergo many edits to incorporate community feedback. If a proposal has enough support, it is merged, meaning that work may begin to integrate the functionality into the codebase.
- Deprecations: The ember-cli informs developers of upcoming changes to the API via deprecation warnings. To see these warnings and Deprecation Guides instructions about how to resolve them, make sure you have the latest CLI. You can use newer CLI versions with older Ember app versions. It's a good idea to resolve deprecations as they arise, so that when new major versions are released, there is less overhead to upgrade.
- Codemods: When it's time for a major version upgrade, and there are syntax edits to make, tools like codemods are provided to help you make changes quickly.
- LTS lifecycle: If you won't be updating your app versions frequently, consider using the Long Term Support (LTS) versions of Ember. These are usually every 4 minor releases (for example, 2.8, 2.12, 2.16) and final major releases (2.18). LTS versions have extended windows for bugfixes and security updates.
- Release schedule: Every six weeks, candidate features are reviewed and either deemed ready to be included in the stable release, or carry on in beta for another cycle until the next release happens.

In short, the best way to make sure you have a healthy application is to comment on RFCs, use an up-to-date version of the CLI, keep your application clean of deprecations, and when possible test your application against beta and canary releases of Ember.

### CODE REVIEW

File: source/learn.html.erb [hunk 1]
```diff
@@ -116,25 +116,52 @@ responsive: true
     <section>
       <h2 id="faq">Frequently Asked Questions</h2>
 
-      <h3 id="faq-future-proof">How do I future-proof my Ember.js application?</h3>
+      <h3 id="faq-glimmer">What is Glimmer? Do I need to learn it too?</h3>
 
       <p>
-        Ember.js has a strong emphasis on <a href="https://speakerdeck.com/wycats/stability-without-stagnation-lessons-learned-from-shipping-ember-dot-js">Stability without Stagnation</a>, which means that framework developers take great care when designing new functionality to make sure it fits with the existing functionalities, and provides a path forward for users to upgrade without being subjected to churn.
+        Glimmer is a component library used in Ember. However, in order to build Ember apps, you don't need to know anything about Glimmer, other than it's super fast and that the contributors to the core codebase love to geek out about it. It's all behind the scenes. If you're curious to know how the Glimmer project fits into the roadmap for Ember's development, you can <a href="https://discuss.emberjs.com/t/readers-questions-will-breaking-up-ember-remove-the-need-for-glimmer-js/14325">learn more here</a>.
```
Comment (MelSumner) [inline source/learn.html.erb]: I think Glimmer is the rendering engine in Ember, not a component library.

Comment (jenweber) [reply source/learn.html.erb]: I had rendering engine at first and then I googled Glimmer and changed it. I'll put it back.

**Comment by jenweber** [comment]
@MelSumner I fixed the Glimmer description.
